lib/createTables.js

```diff
@@ -48,13 +49,13 @@ internals.waitTillActive = (model, callback) => {
 
         status = data.Table.TableStatus;
 
-        setTimeout(callback, 1000);
+        setTimeout(callback, options.pollingInterval || 1000);
       });
     },
   () => status !== 'ACTIVE',
   err => callback(err));
 };
 
 module.exports = (models, config, callback) => {
-  async.eachSeries(_.keys(models), (key, callback) => internals.createTable(models[key], config[key], callback), callback);
+  async.eachSeries(_.keys(models), (key, callback) => internals.createTable(models[key], config.$dynogels, config[key], callback), callback);
```

Comment:
I chose that character because it's an invalid character in a DynamoDB table name, and attributes in this options object are supposed to be table names, where the value is extra data for the DynamoDB CreateTable call.

Basically, using `$` avoids any potential naming conflicts with DynamoDB tables.  We could possibly consider using a symbol instead, if that seems cleaner.